recipes/nbexamples/meta.yaml

```diff
@@ -9,7 +9,7 @@ package:
 source:
   fn: {{ gh_repo }}-v{{ version }}.tar.gz
   url: https://github.com/{{ gh_org }}/{{ gh_repo }}/archive/v{{ version }}.tar.gz
-  sha256: 96926c36333e57aee3820722f1107fbf6a3df9113b8585b14e572fccce385923
+  sha256: f391b42ea6856d92a06fafa18ede464e400c9b2804a2a022d0b7197fd4bd4061
```

Comment:
It seems GitHub is now remaining tarball directories from the original format `<tag>.tar.gz` to `<reponame>-<tag>.tar.gz`. So maybe in rolling out these changes they regenerated the archives resulting in updated timestamps and such. This could cause a checksum change.

To confirm nothing malicious has occurred (not that I think it would, but I wanted to be sure). I tried to compare an archive of my own creation. Here are the steps I used to make it.

```bash
$ git clone https://github.com/danielballan/nbexamples
$ git archive v0.2.0 > nbexamples-v0.2.0.tar.gz
```

After generating this, I downloaded the archive from GitHub. As the binaries of course differ (timestamps), I extracted both into two separate directories and then ran `diff -r`. FWICT there are no differences between the two. Now this doesn't rule out something happening to the repo, but it does verify that the archive can be generated from the repo with no code/binary differences (beyond timestamp changes).

If you wish to be extra sure, someone that cloned/had the repo before this change could repeat the same procedure to determine if there is any difference before and after.

Also, I have emailed GitHub asking for clarification about whether anything changed about GitHub tag archive releases. Will let everyone know what they say.